docs/source/TestParameters.rst

```diff
@@ -244,7 +244,7 @@ Varianter plugins
 
 A plugin interface that can be used to build custom plugins which
 are used by `Varianter`_ to get test variants. For inspiration see
-:class:`avocado.plugins.yaml_to_mux.YamlToMux` which is in-core
+:class:`avocado_varianter_yaml_to_mux.YamlToMux` which is in-core
```

Comment:
"is **in-core**" would be actually false now.

Also, this will break on readthedocs.org, since it won't run "develop" on the optional plugins, and thus, the references to those modules won't be found.  Build logs from rtd:

```
/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/checkouts/latest/docs/source/api/optional-plugins/avocado_resultsdb/avocado_resultsdb.rst:7: WARNING: autodoc: failed to import module u'avocado_resultsdb'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/envs/latest/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 551, in import_object
    __import__(self.modname)
ImportError: No module named avocado_resultsdb
/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/checkouts/latest/docs/source/api/optional-plugins/avocado_robot/avocado_robot.rst:7: WARNING: autodoc: failed to import module u'avocado_robot'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/envs/latest/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 551, in import_object
    __import__(self.modname)
ImportError: No module named avocado_robot
/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/checkouts/latest/docs/source/api/optional-plugins/avocado_runner_docker/avocado_runner_docker.rst:7: WARNING: autodoc: failed to import module u'avocado_runner_docker'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/envs/latest/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 551, in import_object
    __import__(self.modname)
ImportError: No module named avocado_runner_docker
/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/checkouts/latest/docs/source/api/optional-plugins/avocado_runner_remote/avocado_runner_remote.rst:7: WARNING: autodoc: failed to import module u'avocado_runner_remote'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/envs/latest/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 551, in import_object
    __import__(self.modname)
ImportError: No module named avocado_runner_remote
/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/checkouts/latest/docs/source/api/optional-plugins/avocado_runner_vm/avocado_runner_vm.rst:7: WARNING: autodoc: failed to import module u'avocado_runner_vm'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/envs/latest/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 551, in import_object
    __import__(self.modname)
ImportError: No module named avocado_runner_vm
/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/checkouts/latest/docs/source/api/optional-plugins/avocado_varianter_yaml_to_mux/avocado_varianter_yaml_to_mux.rst:7: WARNING: autodoc: failed to import module u'avocado_varianter_yaml_to_mux'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/avocado-framework-ldoktor/envs/latest/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 551, in import_object
    __import__(self.modname)
ImportError: No module named avocado_varianter_yaml_to_mux
```

And results can be seen here:

http://avocado-framework-ldoktor.readthedocs.io/en/latest/TestParameters.html#varianter-plugins